Wangscape/OptionsPaths.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+#include <string>
+class OptionsPaths
+{
+public:
+    virtual ~OptionsPaths();
+    void initialise(std::string filename);
```

Comment:
`Options` stores an `OptionsPaths` value. The `Options` instance is constructed in a spotify-json function, and the `OptionsManager` can only initialise the `OptionsPaths` after the `Options` and `OptionsPaths` constructors have been called.

I could move the `initialise` code into a constructor, but it seems pointless to construct an `OptionsPaths` and then overwrite it with another one. It doesn't make much sense to wrap it in a `boost::optional` or `std::unique_ptr`, either.

I think it's worth making `ModuleGroupContainer::moduleGroupDescription` private and moving assignments ([link](https://github.com/serin-delaunay/Wangscape/blob/json-noise-modules/Wangscape/noise/ModuleManager.cpp#L14)) into a constructor, though.